contrail_api_cli/commands/batch.py

```diff
@@ -0,0 +1,43 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+import shlex
+import sys
+import fileinput
+from six import text_type
+
+from keystoneauth1.exceptions.http import HttpError, HTTPClientError
+
+from ..command import Command, Arg, Option
+from ..manager import CommandManager
+from ..context import Context
+from ..schema import SchemaError
+from ..exceptions import CommandError, CommandNotFound, NotFound, Exists
+from ..utils import printo
+
+
+class Batch(Command):
+    """ Execute commands from file(s) or from stdin
+    """
+    description = "Run commands from a batch file/stdin"
+    files = Arg(nargs="*", help="Name of the batch file")
```

Comment:
I guess this is not only the name but paths to batch files since there can be more than one

Reply:
Yes, right. I changed the description and the help string.